src/components/services/brickService.ts

```diff
@@ -73,10 +73,6 @@ export function canEditBrick(brick: any, user: User) {
   if (isAdmin) {
     return true;
   }
-  let isEditor = checkEditor(user.roles);
-  if (isEditor) {
-    return true;
-  }
```

Comment:
@william-gooch not sure about this. is it needed?

Reply:
Only the editor that is assigned to the brick should be able to edit it, so this is no longer valid.